packages/openactive-integration-tests/test/features/authentication/dynamic-client-registration/feature.json

```diff
@@ -2,8 +2,8 @@
   "category": "authentication",
   "identifier": "dynamic-client-registration",
   "name": "Dynamic Client Registration for Multiple Seller Systems",
-  "description": "For multi-database booking systems where the customer manages Open Booking API client credentials.",
-  "explainer": "",
+  "description": "Implements OpenID Connect Dynamic Client Registration. For multi-database booking systems where the customer manages Open Booking API client credentials.",
+  "explainer": "The Booking System must implement OpenID Connect Dynamic Client Registration, which requires an Initial Access Token",
```

Comment:
It looks like Initial Access Token is just one way of using Dynamic Client Registration. So I wanted to make it clear that that is all that is supported at present